modules/mednist_GAN_workflow_array.ipynb

```diff
@@ -167,20 +143,19 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Download dataset\n",
+    "## Download dataset\n",
     "\n",
     "Downloads and extracts the dataset.\n",
     "\n",
     "The MedNIST dataset was gathered from several sets from [TCIA](https://wiki.cancerimagingarchive.net/display/Public/Data+Usage+Policies+and+Restrictions),\n",
-    "[the RSNA Bone Age Challenge](http://rsnachallenges.cloudapp.net/competitions/4),\n",
+    "[the RSNA Bone Age Challenge](https://www.rsna.org/education/ai-resources-and-training/ai-image-challenge/rsna-pediatric-bone-age-challenge-2017),\n",
     "and [the NIH Chest X-ray dataset](https://cloud.google.com/healthcare/docs/resources/public-datasets/nih-chest).\n",
     "\n",
     "The dataset is kindly made available by [Dr. Bradley J. Erickson M.D., Ph.D.](https://www.mayo.edu/research/labs/radiology-informatics/overview) (Department of Radiology, Mayo Clinic)\n",
     "under the Creative Commons [CC BY-SA 4.0 license](https://creativecommons.org/licenses/by-sa/4.0/).\n",
     "\n",
     "\n",
-    "If you use the MedNIST dataset, please acknowledge the source, e.g.  \n",
-    "https://github.com/Project-MONAI/tutorials/blob/main/2d_classification/mednist_tutorial.ipynb."
+    "If you use the MedNIST dataset, please acknowledge the source, e.g."
```

Comment:
The sentence is incomplete. Maybe delete the "e.g."
If you use the MedNIST dataset, please acknowledge the source.

Thanks!